crypto/pemutil/pkcs8.go

```diff
@@ -292,7 +292,7 @@ func MarshalPKCS8PrivateKey(key interface{}) ([]byte, error) {
 // key derived using PBKDF2 over the given password.
 func DecryptPEMBlock(block *pem.Block, password []byte) ([]byte, error) {
 	if block.Headers["Proc-Type"] == "4,ENCRYPTED" {
-		// nolint
+		//nolint:staticcheck // only used for encryption, not decryption.
```

Comment:
Well, just the opposite, just decryption.

Reply:
oops